public/wp-content/themes/glific/src/scss/variables.scss

```
@@ -4,6 +4,7 @@ $theme-dark-green: #05400A;
 $theme-pewter: #93A29B;
 $theme-mine-shaft: #3B3B3B;
 $theme-gin: #E6F1E7;
+$theme-gray: #F7F7F7;
```

I saw we already have a class with this name, Please update according.

```suggestion
$theme-white-smoke: #f7f7f7;
```